apps/site/src/app/admin/layout/AdminLayout.tsx

```diff
@@ -30,19 +33,31 @@ function AdminLayout({ children }: PropsWithChildren) {
 	const authorized = isAdminRole(role);
 
 	if (!loggedIn) {
-		redirect("/login");
+		router.push("/login");
 	} else if (!authorized) {
-		redirect("/unauthorized");
+		router.push("/unauthorized");
```

Comment:
Thought: Kind of surprised the server-side redirect didn't cause any issues on the deployment.